guides/v2.1/cloud/env/variables-global.md

```diff
@@ -0,0 +1,72 @@
+---
+group: cloud-guide
+title: Global variables
+functional_areas:
+  - Cloud
+  - Configuration
+---
+
+The following _global_ variables control actions in the build, deploy, and post-deploy stages of the `.magento.env.yaml` file. Because global variables impact every stage, you must set them in the `global` stage. Insert these variables in the `global` stage of the `.magento.env.yaml` file:
+
+```yaml
+stage:
+  global:
+    GLOBAL_VARIABLE_NAME: value
+```
+### `MIN_LOGGING_LEVEL`
+
+-  **Default**—_Not set_
+-  **Version**—Magento 2.1.4 and later
+
+Overrides the minimum logging level for all output streams without making changes to the code. This helps to improve troubleshooting problems with deployment. For example, if your deployment fails, you can use this variable to increase the logging granularity globally. See [Set up notifications—Log levels]({{ page.baseurl }}/cloud/env/setup-notifications.html#log-levels). The `min_level` value in Logging handlers overwrites this setting.
+
+```yaml
+stage:
+    global:
+        MIN_LOGGING_LEVEL: debug
+```
+### `SCD_ON_DEMAND`
+
+-  **Default**—_Not set_
+-  **Version**—Magento 2.1.4 and later
+
+Enable generation of static content when requested by a user. This is ideal for the development and testing workflow, because it decreases the deployment time.
+
+Pre-loading the cache using the [`post_deploy` hook]({{ page.baseurl }}/cloud/project/project-conf-files_magento-app.html#hooks) reduces site downtime. The cache warming is available only for Pro projects that contain Staging and Production environments in the Project Web UI and for Starter projects. Add the `SCD_ON_DEMAND` environment variable to the `global` stage in the `.magento.env.yaml` file:
+
+```yaml
+stage:
+  global:
+    SCD_ON_DEMAND: true
+```
+
+The `SCD_ON_DEMAND` variable skips the SCD and the `STATIC_CONTENT_SYMLINK` in both phases (build and deploy), clears the `pub/static` and `var/view_preprocessed` folders, and writes the following to the `app/etc/env.php` file:
```

Comment:
make STATIC_CONTENT_SYMLINK  a link to https://devdocs.magento.com/guides/v2.2/cloud/env/variables-deploy.html#static_content_symlink ?

General observation -- I know SCD is a known acronym but I always find it hard to understand the descriptions when we use too many acronyms.  Would it make sense sometimes to spell it out for clarity -- especially when explaining how a process works .

The `SCD_ON_DEMAND` variable skips deploying static content and generating static content symlinks (``STATIC_CONTENT_SYMLINK`) in both the build and deploy phase, clears the ....

Also-- I find myself wondering if the skipping part is related to the "SKIP_SCD" variable.

I know that these mysteries will be cleared up when you have the SCD doc updates ready, but these are current thoughts on current content that you might be able to tweak in the interim.

Reply:
Added first occurrence.
I know SCD concepts are enigmatic, somewhat. Clarification is on the horizon.